portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,10 +61,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         commentList.add(comment);
     }
 
-    if (numOfComments > commentList.size()) numOfComments = commentList.size();
+    if (commentList.size() > numOfComments) numOfComments = commentList.size();
```

Comment:
Isn't `numOfComments` supposed to be the number of comments the user is asking to see? Usually, that means at most that many comments at a time.

What this if check is doing is increasing the number of comments returned to the front end to the full number of comments.

As an example, say `numOfComments` as entered by the user is 5. Say you have stored 100 comments. This if statement would cause `numOfComments` to be 100.